api/dbadapters/bigquery.ts

```diff
@@ -256,6 +258,61 @@ export class BigQueryDbAdapter implements IDbAdapter {
     await this.runQuery(updateQuery);
   }
 
+  public async setMetadata(action: dataform.IExecutionAction): Promise<void> {
+    const { target, actionDescriptor } = action;
+
+    await this.pool
+      .addSingleTask({
+        generator: async () => {
+          try {
+            const metaData = await this.getMetadata(target);
+            const schemaWithDescription = addDescriptionToMetadata(actionDescriptor?.columns, metaData.schema.fields)
+            
+            const table = await this.getClient(target.database)
+              .dataset(target.schema)
+              .table(target.name)
+              .setMetadata({
+                description: actionDescriptor.description,
+                schema: schemaWithDescription
+              });
+            return table;
+          } catch (e) {
+            if (e && e.errors && e.errors[0] && e.errors[0].reason === "notFound") {
+              // if the table can't be found, just return null
+              return null;
```

Comment:
this seems weird. if this code is being called and this case happens, something has gone Really Wrong - I don't think we should swallow this error

Reply:
Far enough, this was a copy paste from `getMetadata`